### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by AndreyKozhev**
Update Russian translations in the app, common, xml-inflatter modules.

### CODE REVIEW

File: xml-inflater/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -18,5 +18,5 @@
 <resources>
     <string name="msg_cannot_create_view">Не удается создать представление для тега: %s</string>
     <string name="err_no_attr_adapter">Не найдена реализация адаптера атрибутов для %s</string>
-    <string name="msg_view_not_supported">Просмотр %s в настоящее время не поддерживается.</string>
+    <string name="msg_view_not_supported">Предпросмотр %s в настоящее время не поддерживается.</string>
```
Comment (itsaky) [inline xml-inflater/src/main/res/values-ru/strings.xml:21]: Translating `Предпросмотр` says `Preview`. If this is true, then the translation is incorrect. The sentence here should actually say `View %s is not currently supported`.

Need confirmation on this.

Comment (AndreyKozhev) [reply xml-inflater/src/main/res/values-ru/strings.xml:21]: Sorry, I made a mistake. I'll fix it now.

Comment (AndreyKozhev) [reply xml-inflater/src/main/res/values-ru/strings.xml]: Ready!

**Review by itsaky — APPROVED** [review]

**Comment by TimScriptov** [comment]
https://github.com/itsaky/AndroidIDE/blob/0a3e7f28c6a3b3de312239266337e5c2153e514b/app%2Fsrc%2Fmain%2Fres%2Fvalues-ru%2Fstrings.xml#L233

**Comment by TimScriptov** [comment]
Какой смысл несёт последнее предложение?

**Pull request merged**